src/components/ComboBox/ComboBox.tsx

```diff
@@ -44,9 +53,23 @@ export class ComboBox extends React.Component<ComboBoxProps> {
             results = examples.map((option: any) => (
               <Menu.Item
                 key={option.url}
-                onClick={(e: any) => this.setMedia(e, { value: option.url })}
+                onClick={(e: any) =>
+                  this.setMediaAndClose(e, { value: option.url })
+                }
               >
-                {option.url}
+                <div style={{ display: 'flex' }}>
+                  {option.url}
+                  <div style={{ marginLeft: 'auto' }}>
```

Comment:
examples don't really need the button? make an add to playlist for current video?